thoth/worker/tasks/github.py

```diff
@@ -47,20 +41,45 @@ def get_project_repo_github(self, package_name: str):
         """Get project and repo for the given package based on aggregated package info from Thoth's Prescriptions."""
         package_prefix = package_name[:2] + "_"
         headers = {"Authorization": GITHUB_TOKEN}
-
-        gh_link_yaml = requests.get(
-            os.path.join(
-                _PRESCRIPTIONS_BASE_URL, package_prefix, package_name, "gh_link.yaml"
-            ),
-            headers=headers,
+        prescription_link = os.path.join(
+            _PRESCRIPTIONS_BASE_URL, package_prefix, package_name, "gh_link.yaml"
         )
 
+        gh_link_yaml = requests.get(prescription_link, headers=headers)
+
         if gh_link_yaml.status_code != 200:
             _LOGGER.debug(
-                f"{gh_link_yaml.status_code}: Cannot access prescription YAML file"
+                f"Request status code {gh_link_yaml.status_code}: Cannot access prescription YAML file at {prescription_link}"
+            )
+            package_prefix_alternatives = [
+                package_name[:1] + "_",
+                package_name[:3] + "_",
```

Comment:
I'm not sure if I read this correctly. Package names are encoded in the prescriptions repository in the following way:

* if the length of a package name is <=2, then prescriptions are available at `prescriptions/{package_name}>`
* if the length of a package name is >2, the prescriptions are available at `prescriptions/{package_name[:2]}/{package_name}`

Reply:
You are right, I got confused between prefixes and package names when those have a size <2, thanks :+1: